src/main/java/fr/insee/lunatic/conversion/JSONCleaner.java

```diff
@@ -63,4 +66,70 @@ public String preProcessJson2XML(String json) {
 				.replaceAll("<", "&lt;")
 				.replaceAll(">", "&gt;");
 	}
+
+	/**
+	 * Given a Lunatic JSON flat questionnaire, replace the source/target fields by key/values in "symLinks" attribute
+	 * in "PairwiseLinks" components to be compliant with the Lunatic JS library. */
+	public String symLinksCleaning(String stringQuestionnaire) throws IOException {
```

Comment:
https://github.com/InseeFr/Lunatic-Model/pull/79/commits/018f60844955150b77dff13e1aace42dbe4819a7
https://github.com/InseeFr/Lunatic-Model/pull/79/commits/618a20b8f0372c3a0f97b6a166b4734cbaaa19c9
https://github.com/InseeFr/Lunatic-Model/pull/79/commits/80ce536a13024452742705cc2696d8b923761cba

Refactored in smaller parts.

Implementation is actually a bit clumsy, but there is actually no real complexity in it (copying json and editing only the symlink parts). Tests of clean method would break if the symlinks cleaning got wrong, so I think it should be enough for now. We could write more unit test using files that are already in resources, but this implementation will not live long enough to make it worthwhile imo.